src/components/RequestDashboard/SettingsSection.jsx

```diff
@@ -251,6 +253,25 @@ const SettingsSection = props => {
                   </Grid>
                 </React.Fragment>
               );
+            case 'toggle':
+              return (
+                <React.Fragment key={key}>
+                    <Grid key={key} item xs={6} sx={{display: 'flex', alignItems: 'center', gap: '5px'}}>
+                      <p>Hook to send when selecting a patient: </p>
+                      <ToggleButtonGroup
```

Comment:
The toggle is a bit out of place in its current spot.  I have three possible suggestions:

1) Make it a dropdown and style it so it fits in with the rest of the text inputs

2) Make it a checkbox and add it in with the checkboxes

3) Keep it as a toggle and make the checkboxes into toggles, then group them all together